lib/firebase-adapter.js

```diff
@@ -89,30 +97,14 @@ firepad.FirebaseAdapter = (function (global) {
       return;
     }
 
-    if (this.userRef_) {
-      this.userRef_.child("cursor").remove();
-      this.userRef_.child("color").remove();
-    }
-
     this.ref_ = null;
     this.document_ = null;
     this.zombie_ = true;
   };
 
   FirebaseAdapter.prototype.setUserId = function (userId) {
-    if (this.userRef_) {
-      // Clean up existing data.  Avoid nuking another user's data
-      // (if a future user takes our old name).
-      this.userRef_.child("cursor").remove();
```

Comment:
I see that where we are no longer removing the `color` property of the user on disconnect is intentional. Are there any ramifications for also no longer removing the `cursor` property? Curious if that will have any odd behaviors. I haven't noticed any while testing locally, but figured I'd ask.

Reply:
Ah good catch, text can get stuck highlighted if a user selects some text and then exits the pad. We could continue to just clear their cursor when they disconnect like we were doing before, which means your cursor could momentarily disappear when closing a tab. That's so minor though, I doubt anyone would even notice.

Reply:
I'd vote for avoiding the perma-highlighted text. If clearing the cursor on disconnect has a negligible effect of momentarily disappearing when disconnecting from the pad, then that seems like an ok tradeoff.

To add to that scenario: User A is connected to the pad in two different tabs, and closes one of the tabs. Does clearing the cursor on disconnect affect their cursor in the other tab they had open? And if so, in what way?

Reply:
Yeah, I published a new beta release that doesn't remove the cursor, forgot to push that change up though.

Yes, when they have two tabs open and close one, their cursor will disappear until they click in the pad again, at which point the cursor will re-appear for everyone in the new location.